homeassistant/components/mythicbeastsdns.py

```diff
@@ -0,0 +1,58 @@
+"""
+Integrate with Mythic Beasts Dynamic DNS service.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/mythicbeastsdns/
+"""
+from datetime import timedelta
+import logging
+import voluptuous as vol
+import mbddns
```

Comment:
Imports from component or platform requirements have to be done within functions, since requirements will be installed during runtime.

Please move the import of `mbddns` to the functions that use it.